view_options/app/build.gradle.kts

```
@@ -69,12 +69,12 @@ dependencies {
     implementation("com.github.bumptech.glide:glide:4.13.2")
 
     // Firework SDK
-    val fireworkSdkVersion = "6.0.0-beta.1"
-    implementation("com.github.loopsocial.firework_sdk_v2:fireworkSdk:$fireworkSdkVersion")
+    val fireworkSdkVersion = "6.0.0-beta.2-SNAPSHOT"
```

❓Doesn't "6.0.0-beta.2" works here?
```suggestion
    val fireworkSdkVersion = "6.0.0-beta.2"
```

In general, we should not rely that much on `-SNAPSHOT` releases.